daemon/config_unix.go

```diff
@@ -89,6 +91,7 @@ func (config *Config) InstallFlags(flags *pflag.FlagSet) {
 	flags.Int64Var(&config.CPURealtimePeriod, "cpu-rt-period", 0, "Limit the CPU real-time period in microseconds")
 	flags.Int64Var(&config.CPURealtimeRuntime, "cpu-rt-runtime", 0, "Limit the CPU real-time runtime in microseconds")
 	flags.StringVar(&config.SeccompProfile, "seccomp-profile", "", "Path to seccomp profile")
+	flags.Int64Var(&config.ShmSize, "shm-size", defaultShmSize, "Set the default shm size for containers")
```

Comment:
So, should it be `default-shm-size`? I think it's better, otherwise it could also mean the shared memory size for the docker daemon itself. So maybe `default-shm-size` is a better option, despite the concern raised by @thaJeztah regarding the proliferation of `default-*` options.